### PR TITLE
Update CGC membership list

### PR DESCRIPTION
**Pull request by AevaOnline**

### CODE REVIEW

**Review by CodeMonkeyLeet — APPROVED** [review]
LGTM, thanks for taking the initiative, Aeva!

**Review by dthaler — APPROVED** [review]

File: docs/Community/governance/README.md [hunk 1]
```diff
@@ -19,16 +19,23 @@ Members
...
+| Anand Krishnamoorthi | Microsoft   | anakrish@microsoft.com        | anakrish       |
+| Dave Thaler          | Microsoft   | dthaler@microsoft.com         | dthaler        |
+| Radhika Jandhyala    | Microsoft   | radhikaj@microsoft.com        | radhikaj       |
+| Simon Leet           | Microsoft   | simon.leet@microsoft.com      | CodeMonkeyLeet |
+
+
+Emeritus Members
+----------------
+
+| Name                 | Company     | Email                         | GitHub Alias   |
```
Comment (radhikaj) [inline docs/Community/governance/README.md:32]: Can we also add Jordan here?

Comment (radhikaj) [reply docs/Community/governance/README.md:32]: We can also do it in a different PR

**Review by johnkord — APPROVED** [review]

**Review by anakrish — APPROVED** [review]

File: docs/Community/governance/README.md (hunk 1 above)
Comment (AevaOnline) [reply docs/Community/governance/README.md:32]: My intent was only to move those members who replied on the CGC list.

I have no concerns about adding Jordan, but for tracking purposes, I'd suggest a separate commit / PR.

**Pull request merged**